doc/ref_model/chapters/chapter07.rst

```diff
@@ -394,52 +394,116 @@ ZTA principles applied to Cloud infrastructure components are the following:
-
-**Vulnerability identification**
-
-Vulnerability management must be continuous: from development to runtime, not only on the development process, but during all the life of the application or workload or service. When a public vulnerability on a component is released, the update of the component must be triggered. When an SBOM recording the code composition is provided, the affected components will be easier to identify. It is essential to remediate the affected components as soon as possible, because code transparency can also be exploited by attackers who can take the benefit of vulnerabilities.
-
-The CVE and the CVSS must be used to identify vulnerabilities and their severity rating. The CVE identifies, defines, and catalogues publicly disclosed cybersecurity vulnerabilities while the CVSS is an open framework to calculate the vulnerabilities' severity score.
+Vulnerability identification
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Various images scanning tools, such as Clair or Trivy, are useful to audit images from security vulnerabilities. The results of vulnerabilities scan audit must be analysed carefully when it is applied to vendor offering packaged solutions; as patches are not detected by scanning tools, some components can be detected as obsolete.
+Vulnerability management must be continuous: from development to runtime, not only on the development process, but during all 
+the life of the application or workload or service. When a public vulnerability on a component is released, the update of the 
+component must be triggered. When an SBOM recording the code composition is provided, the affected components will be easier 
+to identify. It is essential to remediate the affected components as soon as possible, because the vulnerability can be 
+exploited by attackers who can take the benefit of code weakness.
 
-**Trusted repositories**
+The CVE and the CVSS must be used to identify vulnerabilities and their severity rating. The CVE identifies, defines, and 
+catalogues publicly disclosed cybersecurity vulnerabilities while the CVSS is an open framework to calculate the 
+vulnerabilities' severity score.
 
-A dedicated internal isolated repository separated from the production environment must be used to store vetted open source content, which can include images, but also installer and utilities. These software packages must be signed and the signature verified prior to packages or images installation. Access to the repository must be granted by a dedicated authorization. The code must be inspected and vulnerabilities identified as described previously. After validating the software is risk free, it can be moved to the appropriate production repository.
+Various images scanning tools, such as Clair or Trivy, are useful to audit images from security vulnerabilities. 
```

Comment:
I would say "Various images scanning tools, including open-source tools like Clair or Trivy, are useful to audit images from security vulnerabilities. ", to inform the reader about open-source tools available.  There are plenty of proprietary tools of course.

Reply:
Proposal added